src/constants.js

```diff
@@ -90,6 +90,7 @@ export const FIELD_DATE: 'date' = 'date';
 export const FIELD_NAME: 'name' = 'name';
 export const FIELD_TYPE = 'type';
 export const FIELD_SIZE: 'size' = 'size';
+export const FIELD_PROMOTED_BY = 'promoted_by';
```

Comment:
also near the other `[...]_BY`

Reply:
Good catch, didn't see this sorting. Thought it was random. Kinda weird way to order these. Would prefer alphabetically.